modules/osdk-hh-project-layout.adoc

```diff
@@ -47,7 +47,7 @@ The Hybrid Helm Operator scaffolding is customized to be compatible with both He
 |`api/`
 |Contains the Go API definition.
 
-|`controllers/`
+|`internal/controllers/`
```

Comment:
Just noting that it renders fine in the preview:

![image](https://github.com/user-attachments/assets/637486a6-3dee-4011-9e20-ee70758b675e)